templates/api/knowledge-panels/health/ingredients/ingredients.tt.json

```diff
@@ -3,13 +3,26 @@
     "topics": [
         "health"
     ],
-    "elements": [   
+    "type": "inline",
+    "elements": [
+        [% IF panel.lc != panel.ingredients_text_lc %]
```

Comment:
This condition is not really clear to me. Is that because if lc is same as ingredients_text_lc we already displayed it ? If so, a comment could be helpful.

Reply:
It's because we display something like "Ingredients (Spanish) : leche de vaca" when the language of the ingredients is different from the language of the website / the app. If you are on the French site and we have ingredients in French, we just display "Ingrédients : lait de vache", not "Ingrédients (français) : lait de vache".